src/core/checkedint.d

```diff
@@ -741,4 +763,4 @@ unittest
     assert(mulu(0UL, 0UL, overflow) == 0);
     assert(overflow);                   // sticky
 }
-}
```

Comment:
Line ending problems is why I run my submissions through a line ending fixer (tolf in the D tools).